lib/ncurses/tinfo/Makefile

```diff
@@ -273,16 +273,13 @@ keys.list: MKkeys_list.sh Caps Caps-ncurses
 # Build tools
 DEPENDOBJS+= make_hash make_keys
 build-tools: make_hash make_keys
-BUILD_TOOLS_CFLAGS+=	-I. -I${.CURDIR}/../ncurses
-BUILD_TOOLS_CFLAGS+=	-I${NCURSES_DIR}/ncurses -I${NCURSES_DIR}/include
 
 make_keys: make_keys.c names.c ncurses_def.h ${HEADERS} ${SRCHDRS} ${BUILD_TOOLS_META}
```

Comment:
The addition of SRCHDRS to the dependencies of make_keys and make_hash is also a local diff, but I think it is orthogonal to BUILD_TOOLS_*.  I've no idea if it needs upstreaming?

Reply:
```commit b7a1f7fd70367162ae71d75f17bfe1b12e0fac69
Author: Alex Richardson <Alexander.Richardson@cl.cam.ac.uk>
Date:   Fri Dec 14 21:48:21 2018 +0000

    Minor crossbuilding fixes``` is the source of that